examples/rds/rdsinstance.yaml

```diff
@@ -0,0 +1,33 @@
+# wip, repurpose this attach instances to the dbcluster.
```

Comment:
Yeah, that would be a good idea.

Reply:
I could not get the rdsinstance eg working so I have removed it from here for now. Will continue to try and get it working but we dont want to block this from being merged.
fyi I am using this to test locally  
```
apiVersion: database.aws.crossplane.io/v1beta1
kind: RDSInstance
metadata:
  name: example-rds
spec:
  forProvider:
    allocatedStorage: 20
    autoMinorVersionUpgrade: false
    backupRetentionPeriod: 0
    caCertificateIdentifier: rds-ca-2019
    copyTagsToSnapshot: false
    dbInstanceClass: db.m1.small
    enableIAMDatabaseAuthentication: false
    enablePerformanceInsights: false
    engine: aurora-postgresql
    licenseModel: general-public-license
    multiAZ: true
    port: 3306
    preferredBackupWindow: 06:15-06:45
    preferredMaintenanceWindow: sat:09:21-sat:09:51
    publiclyAccessible: false
    storageEncrypted: false
    storageType: gp2
    dbClusterIdentifier: example-rds
    region: us-east-1
  providerConfigRef:
    name: default
```

and this is the error I get -

```2021-03-02T21:47:02.159-0800	DEBUG	controller-runtime.manager.events	Warning	{"object": {"kind":"RDSInstance","name":"example-rds","uid":"7b31021f-e8f4-4155-9c8b-4dd2b909fe52","apiVersion":"database.aws.crossplane.io/v1beta1","resourceVersion":"375236"}, "reason": "CannotCreateExternalResource", "message": "cannot create RDS instance: InvalidParameterCombination: The requested DB Instance will be a member of a DB Cluster. Set master user password for the DB Cluster."}```